bin/yii

```diff
@@ -41,6 +43,15 @@ use Yiisoft\Yii\Console\Application;
 
     /** @var ContainerInterface $container */
     $container = $params['console']['container']();
```

Comment:
The PR looks great there is no way to avoid the static function in the params-console?

Reply:
> The PR looks great there is no way to avoid the static function in the params-console?

I don't know for what purpose it is in the params, I can move it to the init `yii` script, but I don't sure if it will be correct.

Reply:
It is there on purpose. To be able to use your own container implementation.